### ISSUE TITLE
X0

### ISSUE THREAD

**Issue by ASEMBER2** [Tree-of-Sex/tos-ontology#40]
### In which category is your new term request?

- [X] new_definition
- [X] existing_definition

### Preferred term label

X0

### Synonyms

♂X0/♀XX
X0/XX

### Textual definition

A male-heterogametic sex chromosome system derived from ♂XY/♀XX standard sex chromosome system. It may arise after a Y chromosome completely disappears from the genome (due to nondisjunction in meiosis, as it no longer manages to pair stably with the X chromosome, especially at terminal stages of its degeneration). This may be allowed by selection to be fixed especially if the master sex-determining gene translocated/transposed to X chromosome or new X-linked regulator appeared to goven sex determination, and when haploinsufficient genes and/or genes crucial for male fertility (e.g. fo spermatogenesis) are no longer present on degenerated Y. This situation creates karyotype differences between males and females (male has one chromosome less) which may be confused by X1X2Y multiple sex chromosome composition. The way how to distinguish between the two systems is to analyse the chromosome pairing in the first meiotic division in males (X0 system is manifested by an X univalent among the autosome bivalents, while X1X2Y system can be distinguished based on trivalent structure composed of the three sex chromosomes). X0 sex chromosome system is typical e.g. for majority of orthopteran insects. 

### Suggested parent term

0 as sex chromosome notation
karyotype
genotype
genomic genotype
heterogametic sex genotype
sex determination
sex chromosome GO:0000803

### Attribution

0000-0003-4441-9615

### Additional context
Bearing in mind X0/XX sex chromosome system, X-autosome fusion will create neo-XY sex chromosome system. Reciprocal translocation between X0/XX and autosome pair may generate X1X2Y multiple sex chromosome system. Another situation which may be confused with X0/XX sex chromosome system is X0/X0 sex chromosome system found in some mammals (e.g. Tokudaia osimensis).

Suggested literature:

Kuroiwa, A., Ishiguchi, Y., Yamada, F., Shintaro, A., & Matsuda, Y. (2010). The process of a Y-loss event in an XO/XO mammal, the Ryukyu spiny rat. Chromosoma, 119, 519-526. https://doi.org/10.1007/s00412-010-0275-8

Blackmon, H., & Demuth, J. P. (2015). The fragile Y hypothesis: Y chromosome aneuploidy as a selective pressure in sex chromosome and meiotic mechanism evolution. BioEssays, 37(9), 942-950. https://doi.org/10.1002/bies.201500040

Jonika, M. M., Alfieri, J. M., Sylvester, T., Buhrow, A. R., & Blackmon, H. (2022). Why not Y naught. Heredity, 129(2), 75-78. https://doi.org/10.1038/s41437-022-00543-z

Terao, M., Ogawa, Y., Takada, S., Kajitani, R., Okuno, M., Mochimaru, Y., ... & Kuroiwa, A. (2022). Turnover of mammal sex chromosomes in the Sry-deficient Amami spiny rat is due to male-specific upregulation of Sox9. Proceedings of the National Academy of Sciences, 119(49), e2211574119. https://doi.org/10.1073/pnas.2211574119

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ValenzuelaLab** [comment]
All the information above is great. However, for the definition, I suggest to keep it simple so that it is operational:

A male-heterogametic sex chromosome system of sex determination (XX/X0) in which individuals carrying a single X develop as males and individuals carrying two Xs develop as females.

**Comment by ASEMBER2** [comment]
I agree. The rest/part of the definition can be also moved to "Additional context".
